src/publish_fetch_handler.cpp

```diff
@@ -1,26 +1,113 @@
 // SPDX-FileCopyrightText: Copyright (c) 2025 Cisco Systems
 // SPDX-License-Identifier: BSD-2-Clause
 
-#include <quicr/publish_fetch_handler.h>
+#include "quicr/publish_fetch_handler.h"
+
+#include "quicr/detail/transport.h"
 
 namespace quicr {
     PublishTrackHandler::PublishObjectStatus PublishFetchHandler::PublishObject(const ObjectHeaders& object_headers,
                                                                                 const BytesSpan data)
     {
+        auto transport = GetTransport().lock();
+
+        if (!transport) {
+            return PublishObjectStatus::kInternalError;
+        }
+
         bool is_stream_header_needed{ !sent_first_header_ };
         sent_first_header_ = true;
-        if (publish_object_func_ == nullptr) {
-            return PublishObjectStatus::kInternalError;
+
+        if (!GetRequestId().has_value()) {
+            return PublishTrackHandler::PublishObjectStatus::kNoSubscribers;
+        }
+
+        ITransport::EnqueueFlags eflags;
+
+        std::uint64_t group_id = object_headers.group_id;
+        std::uint64_t object_id = object_headers.object_id;
+        std::uint16_t ttl = object_headers.ttl.has_value() ? object_headers.ttl.value() : default_ttl_;
+        std::uint8_t priority =
+          object_headers.priority.has_value() ? object_headers.priority.value() : default_priority_;
+
+        object_msg_buffer_.clear();
+
+        switch (default_track_mode_) {
+            case TrackMode::kDatagram: {
```

Comment:
This appears to be a cut/paste of SendObject from transport. Makes sense when it was common code. Now that this is moved to each handler, this doesn't make sense here. 

Fetch does not use datagram.  Fetch does not use more than one stream.